src/components/LanguageSelector.js

```diff
@@ -31,7 +35,7 @@ const LanguageSelector = () => {
   return (
     <div className="popup">
       <Button aria-describedby={id} onClick={toggleLanguage}>
-        <LanguageIcon style={{ fill: 'white', zIndex: '99999' }} />
+        <LanguageIcon className={classes.icon} />
```

Comment:
this element is still unaligned and the solution is not changing the whole icon class as it would affect the other icons